courses/machine_learning/deepdive2/structured/labs/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Consider renaming the OUTPUT_DIR so both outputs from the previous labs' DNN model and the wide and deep model here could be saved. </p><p> </p><p>Not necessary, but could be cool to be able to show both training/eval curves side by side in Tensorboard (when that is working). </p><p> </p><p>note: this could also require chaning the OUTPUT_DIR in the DNN model notebook and solution. Or could just leave as is there and modify here. </p><p> </p><p>(if changed here, should be reflected in the solution notebook as well)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Good idea, originally I intended for the students to choose either to build a DNN or a Wide and Deep model and would only do one of the notebooks, but this allows them to do both.</p>

Reply:
ok Great!